### PR TITLE
Add helper functions for placing ships on the board

### PR DESCRIPTION
**Pull request by Safisyx**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -11,4 +11,5 @@ ReactDOM.render(
     <App />
   </Provider>,
   document.getElementById('root')
+)
```
Comment (Ju1111) [inline src/index.js:14]: good one! didn't see that!

File: src/lib/functions.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
+//(x,y) will be coordinates of a square
+export const free = (board, x,y) => {
+  return board[x][y] === '0'
+} // check if a square is free
+
+
+export const rows = (board) => {
```
Comment (Ju1111) [inline src/lib/functions.js:7]: why do we have to define the board?

File: src/lib/functions.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+  return board
+}
+
+// Returns a transposed array of columns on the board
+export const cols = (board) => {
```
Comment (Ju1111) [inline src/lib/functions.js:12]: might be necessary or not.

**Review by Ju1111 — APPROVED** [review]

File: src/lib/functions.js (hunk 2 above)
Comment (Safisyx) [reply src/lib/functions.js:7]: Actually, it's just taking the rows from the board which is in fact the board itself :smiley_cat:

**Pull request merged**